crates/testing/src/sdk.rs

```diff
@@ -194,8 +193,6 @@ fn generate_bindings(language: &str, wasm_file: &str, client_project: &str, gene
         create_dir_all(generate_dir).expect("Error creating generate subdir");
         invoke_cli(&[
             "generate",
-            "--debug",
-            "--skip_clippy",
```

Comment:
it turns out that these args were essentially unused, because we pass `--wasm-file` anyway (skipping the build step). It became an error in this PR because I added a `.conflicts_with("build_options")` to `--wasm-file`.